packages/webapp/components/Governance/ProposalCard.tsx

```diff
@@ -1,14 +1,14 @@
 import { Badge, Box, HStack, StackProps, Text, TextProps, Tooltip, VStack } from "@chakra-ui/react"
 import { FC } from "react"
-import { EffectiveProposalStatus, QuorumVotes, Votes, isFinalized } from "../../utils/governanceUtils"
+import { EffectiveProposalStatus, QuorumVotes, Votes, isFinalized } from "@utils/governanceUtils.ts"
```

Comment:
why add ".ts" at the end?

Reply:
You're right, it's not needed..but the new AI I have integrated with my IDE was telling me there was a problem finding the file without the `.ts` and now it's not...probably just lag. Changing it.